CRS/Cirrus.Node/Controllers/IndexerContractsController.cs

```diff
@@ -70,7 +74,7 @@ public IActionResult GetContractInfo([FromQuery] string txHash)
                 typeName = this.stateRoot.GetContractType(address);
             }
 
-            List<LogResponse> logResponses = null;
+            List<LogResponse> logResponses = new List<LogResponse>();
```

Comment:
This is good and ensures empty array is returned as oppose to null value. Methods and APIs should return empty arrays and not null when there is no results.